src/cloud-api-adaptor/podvm-mkosi/Makefile

```diff
@@ -24,21 +25,18 @@ container: fedora-binaries-builder binaries image-container
 PHONY: fedora-binaries-builder
 fedora-binaries-builder:
 	@echo "Building $(BUILDER) image..."
-ifeq ($(ARCH),s390x)
 	docker buildx build \
 		-t $(BUILDER) \
-		--build-arg ARCH=s390x \
-		--build-arg PROTOC_ARCH=s390x \
-		--build-arg YQ_ARCH=s390x \
-		--build-arg YQ_CHECKSUM=sha256:4e6324d08630e7df733894a11830412a43703682d65a76f1fc925aac08268a45 \
+		--build-arg GO_VERSION=$(GO_VERSION) \
```

Comment:
How was it working earlier without all these new additions ?
These new args doesn't seem to be related to the RUST_VERSION change

Reply:
So my understanding when I reviewed is that these updates allow the podvm_builder versions to be updated from their defaults that are hardcoded in Dockerfile.podvm_builder.fedora, and are set in Makefile.defaults which is included here. Previously we'd just fall back to the hard-coded versions in the Dockerfile, so I assumed that DaLi decided to allow all to be overrided from versions.yaml, rather than just RUST_VERSION. 

I'm not sure if there is an argument to also make the `ARG` values in the Dockerfiles not set so that we'd spot error like this quicker and not have to duplicate versions in multiple places, but that might have more knock-on changes that I don't understand?

Reply:
I see, probably a separate commit with description will be helpful here.

Reply:
Thanks @stevenhorsman yeah, I saw that these values in the podvm builder for packer are all [to be overriden from versions.yaml](https://github.com/confidential-containers/cloud-api-adaptor/blob/main/src/cloud-api-adaptor/Makefile#L170-L179,) so that I make this change to make it same as the packer-ubuntu one.

Reply:
update the commit title to `align the fedora-binaries-builder with podvm-builder`